common/src/main/java/com/wynntils/features/chat/TerritoryDefenseMessageFeature.java

```diff
@@ -18,7 +18,7 @@
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
-@ConfigCategory(Category.UNCATEGORIZED)
+@ConfigCategory(Category.CHAT)
```

Comment:
I disagree. Just because we are sending a message does not mean it belongs to the chat category. This is more to do with Guilds; so if we had a Guild category it would belong there. Maybe if we have other Guild features it would make sense to create one?